lua/starfall/libs_sh/trace.lua

```diff
@@ -24,6 +24,12 @@ SF.RegisterLibrary("trace")
 
 local structWrapper, util_TraceLine, util_TraceHull, util_IntersectRayWithOBB, util_IntersectRayWithPlane, util_Decal, util_PointContents, util_AimVector = SF.StructWrapper, util.TraceLine, util.TraceHull, util.IntersectRayWithOBB, util.IntersectRayWithPlane, util.Decal, util.PointContents, util.AimVector
 
+local vec_meta = getmetatable(Vector(0, 0, 0))
+local vec_SetUnpacked = vec_meta.SetUnpacked
+
+local ang_meta = getmetatable(Angle(0, 0, 0))
+local ang_SetUnpacked = ang_meta.SetUnpacked
```

Comment:
Change vec_meta and ang_meta to a name that doesn't conflict with the ones on line 40-41.

Reply:
I don't think this change will affect anything. the same is done in render lib.